docs/app/pages/components/components-sections/panels/side-panel/side-panel.component.ts

```diff
@@ -1,14 +1,54 @@
 import { Component } from '@angular/core';
 import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
 import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import { IPlunkProvider } from '../../../../../interfaces/IPlunkProvider';
+import { IPlunk } from '../../../../../interfaces/IPlunk';
 
 @Component({
     selector: 'uxd-side-panel',
     templateUrl: './side-panel.component.html',
     styleUrls: ['./side-panel.component.less']
 })
 @DocumentationSectionComponent('ComponentsSidePanelComponent')
-export class ComponentsSidePanelComponent extends BaseDocumentationSection {
+export class ComponentsSidePanelComponent extends BaseDocumentationSection implements IPlunkProvider {
+
+    plunk: IPlunk = {
+        files: {
+            'app.component.html': this.snippets.raw.appHtml,
+            'app.component.ts': this.snippets.raw.appTs,
+            'app.component.css': this.snippets.raw.appCss,
+        },
+        modules: [
+            {
+                imports: ['SidePanelModule'],
+                library: '@ux-aspects/ux-aspects'
+            },
+        ]
+    };
+
+    open = false;
+    inline = false;
+    width = '50%';
+    top = '53px';
+    modal = false;
+    animate = true;
+    closeOnExternalClick = false;
+
+    get attachTo(): string {
+        return this._attachTo;
+    }
+
+    set attachTo(value: string) {
+        this._attachTo = value;
+        if (value === 'window') {
```

Comment:
Yes, but there will most likely be a new documentation section if/when this gets added to the MF site, since the example would probably be based on something from the style guide.